### ISSUE TITLE
.npy weights

### ISSUE THREAD

**Issue by Dayire** [Vladkryvoruchko/PSPNet-Keras-tensorflow#39]
Hello,

Can anyone upload the .npy weights for cityscapes (and others if possible) since the conversion is not working for me, PSPNet caffe is not working well.

Thanks!

**Comment by jmtatsch** [comment]
@Dayire what makes you think the conversion is not working?
Here are some npy files from my dropbox, not sure if they were pre or post #31

https://www.dropbox.com/s/slzgd07ijcyl45b/pspnet50_ade20k.npy?dl=0
https://www.dropbox.com/s/oimz7wrgqhfmkk9/pspnet101_cityscapes.npy?dl=0
https://www.dropbox.com/s/ht3qup5r1nourbg/pspnet101_voc2012.npy?dl=0

Also check out my PSPNet-Keras-tensorflow fork, it should be working.

**Comment by jmtatsch** [comment]
I checked, those should be post batch norm order fix. What is your error exactly?

**Comment by Dayire** [comment]
my error was with PSPNet caffe mostly, even after installing it. It didn't convert correctly. I've deleted it now but I will try to do the step again. If that would lead to a bug?

**Comment by jmtatsch** [comment]
Ah now I understand. Getting a running version of pspnets caffe flavor with the current drivers & cudnn is quite a challenge. But you can directly use the Keras Model files now without conversion from npy.

**Comment by Dayire** [comment]
I'm using this repo to train, for this, I need the numpy weight as I checked in the training code. Or is it possible to change few lines to use the keras weights?

**Comment by jmtatsch** [comment]
It doesn’t make sense to load the weights from npy format every time. That is much slower and maybe only a little bit more universal towards keras changes and python2/3 marshalling issues. Once you have converted the weights to h5 you should use that as starting point for your training endeavors.

**Comment by Dayire** [comment]
I understand now, I'm new to Keras. Thanks!

**Comment by xjNiu** [comment]
The links to .npy files are not working now. Can you update it?@jmtatsch

**Comment by gulzainali98** [comment]
at my end when i run the command 

> python pspnet.py -m pspnet101_cityscapes -i example_images/cityscapes.png -o example_results/cityscapes.jpg

It always runs .npy weights by default. How to solve this issue? i have .h5 weights downloaded.
